HW1/tast1.py

```diff
@@ -0,0 +1,8 @@
+number = int(input("Введите трехзначное число: "))
+if number // 1000 == 0 and number > 100:
```

Comment:
Если этими условиями вы пытаетесь проверить трехзначность числа, то есть более легкий и верный способ:
```suggestion
if 1000 > number > 99:
```